File: Templates/Examples/BasicSamples/VisualBasic/PartialCodeBehind.vb
' Name: Partial Template.
' Author: Blake Niemyjski
' Description: Shows off how to use a code behind as a partial class.

Imports System

Namespace MyCustomNameSpace

    Public Partial Class PartialTemplate
        ' This property uses the property that is declared in the template.
        Public ReadOnly Property PropertyInCodeBehind As String
           Get
                Return String.Format("{0}-Example", PropertyInTemplate)
            End Get
        End Property
    End Class
    
End Namespace

File: trunk/Templates/Examples/BasicSamples/CSharp/PartialCodeBehind.cs
// Name: Partial Template.
// Author: Blake Niemyjski
// Description: Shows off how to use a code behind as a partial class.

using System;

namespace MyCustomNameSpace
{
    public partial class PartialTemplate
    {
        // This property uses the property that is declared in the template.
        public string PropertyInCodeBehind
        {
            get
            {
                return string.Format("{0}-Example", PropertyInTemplate);
            }
        }
    }
}